### PR TITLE
IntelliSense for users to write ECS task definition in their VScode

### PR DESCRIPTION
**Pull request by seongmDevX**
## Description

Added another file match to the jsonValidation for ECS task definition. Users will create a file named `task-def.json` to enable the intellisense or pattern matching the file name `task-def.json`.

## Motivation and Context

This change is for when customers create ECS task definition in their VScode. Currently there is no code completion, documentation look up or error checking. So the ECS DevX team has created a JSON schema that would provide these features. It would improve the customer's experience greatly. 

https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create-task-definition.html

## Related Issue(s)

No related issues.

## Testing

Ran `npm run test` and passed
Pressed F5 and passed

Without IntelliSense:

Screen Shot 2019-08-05 at 11 56 02 AM

With IntelliSense:
Screen Shot 2019-08-05 at 11 54 55 AM

Screen Shot 2019-08-05 at 11 57 17 AM

Screen Shot 2019-08-05 at 12 00 17 PM

My changes are not affecting other areas of the code

## Screenshots (if appropriate)
Screen Shot 2019-07-26 at 2 11 35 PM

## Checklist

- [x ] I have read the **README** document
- [x ] I have read the **CONTRIBUTING** document
- [ x] My code follows the code style of this project
- [ x] I have added tests to cover my changes
- [X ] All new and existing tests passed
- [ x] A short description of the change has been added to the changelog using the script `npm run newChange`

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -331,6 +331,10 @@
             {
                 "fileMatch": ".aws/templates.json",
                 "url": "./out/src/schemas/templates.json"
+            },
+            {
+                "fileMatch": "*task-def.json",
```
Comment (awschristou) [inline package.json]: "task-def.json" seems like too general a file match. I'm concerned about users with non-ECS files getting this schema applied to them. Do you have a more specific file match that you could apply?

File: .changes/next-release/Feature-7eb562a6-6f2d-4b9c-8b70-59dc134cdbd5.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Added JSON validation for ECS task definition intellisense"
```
Comment (awschristou) [inline .changes/next-release/Feature-7eb562a6-6f2d-4b9c-8b70-59dc134cdbd5.json]: @Paul-B-AWS can you provide some User documentation and some extension description revisions around this feature?
(Not in this PR)

File: package.json (hunk 1 above)
Comment (seongmDevX) [reply package.json]: Would `*ecs-task-def.json` would work for you?

File: .changes/next-release/Feature-7eb562a6-6f2d-4b9c-8b70-59dc134cdbd5.json (hunk 2 above)
Comment (seongmDevX) [reply .changes/next-release/Feature-7eb562a6-6f2d-4b9c-8b70-59dc134cdbd5.json]: Yes, we are currently working on one.

File: package.json (hunk 1 above)
Comment (awschristou) [reply package.json]: Sounds good to me

Comment (seongmDevX) [reply package.json]: Sweet and done!

**Pull request merged**